notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/service/jpa/JpaAction.java

```diff
@@ -81,4 +84,9 @@ public void setClazz(String clazz) {
         this.clazz = clazz;
     }
 
+    @Override
+    public String toString() {
+        return "JpaAction [id=" + id + ", label=" + label + ", clazz=" + clazz + "]";
```

Comment:
would [`org.apache.commons.lang3.builder.ToStringBuilder`](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/builder/ToStringBuilder.html) be a more effective way to generate this string?